tardis/plasma/tests/test_nlte_solver.py

```diff
@@ -217,15 +205,8 @@ def test_jacobian_matrix(
         simple_total_col_recomb_coefficients,
     )
 
-    desired_jacobian_matrix = [
-        [-0.07760098, 0.09927163, 0.0, 0.0, 0.0, 0.23467404],
-        [1.0, 1.0, 0.0, 0.0, 0.0, 0.0],
-        [0.0, 0.0, -0.15726292, 0.22196045, 0.0, 0.04022076],
-        [0.0, 0.0, 0.0, -0.8346228, 0.16147935, 0.20061248],
-        [0.0, 0.0, 1.0, 1.0, 1.0, 0.0],
-        [0.0, 1.0, 0.0, 1.0, 2.0, -1.0],
-    ]
-    assert_almost_equal(actual_jacobian_matrix, desired_jacobian_matrix)
+    # TODO: allow for assert_almost_equal
```

Comment:
I think if I understand correctly, `assert_allclose` is good enough

Reply:
I see